docs/products/kafka/howto/keystore-truststore.rst

```diff
@@ -0,0 +1,52 @@
+Creating keystore and truststore for Apache Kafka
+=================================================
+
+Aiven for Apache Kafka utilises TLS (SSL) to secure the traffic between its services and client applications. This means that clients must be configured with the
+right tools to be able to communicate with the Aiven services.
+
+Keystores and truststores are password-protected files accessible to the client that interacts with the service. 
+To create these files:
+
+1. Log in to the `Aiven web console <https://console.aiven.io/>`_ and select your Aiven for Apache Kafka service.
+
+2. Download the **Access Key**, **Access Certificate** and **CA Certificate** 
+
+.. image:: /images/products/kafka/ssl-certificates-download.png
+    :alt: Download the Access Key, Access Certificate and CA Certificate from the Aiven console
+
+3. Save the resulting ``service.key``, ``service.cert`` and ``ca.pem`` files in a specific folder accessible by your clients.
+
+4. In the folder where the certificates are stored, use ``openssl`` utility to create the keystore with the ``service.key`` and
```

Comment:
In a way, those certificates can be stored anywhere. The later generated files should be linked in the client.
So I might even remove this line, instead add an item at the end, that now your clients can refer to these certificates

Reply:
which you already do, maybe change to the step "#. Reference the generated certificates ``client.keystore.p12`` and truststore named ``client.truststore.jks`` in the SSL configuration of your client."

Reply:
I wanted to make the point to save them in an handy folder for further processing, I'll reword